### ISSUE TITLE
Does it delete from git history too?

### ISSUE THREAD

**Issue by bgintzairspace** [DevPrinceK/print_slayer#2]
On the output, I see a lot of 

print/debugPrint Statements Removed in .history/lib/[path_to_files]/file_2024061243.dart

Is that right?  I was hoping I could always go back and look at a previous commit to see print statements if I needed, but this seems like it removes them from the history too??

**Comment by DevPrinceK** [comment]
Yes, if your git history is a directory located inside your main project.
You can ignore files by parsing the name of the files to the -i flag.
I guess I should include a flag for ignoring a directory as well.

Thanks for the feedback. Will update. Sorry for your git history though.

**Comment by bgintzairspace** [comment]
Maybe require an explicit folder that you have to specify to run it on?  That way you can safely run it on lib/ ??
